doc/method/method-details.md

```diff
@@ -380,12 +380,14 @@ are 106 additions in JSON form, 97 additions and patches in tabular
 form, and approximately 121 in Python form.
 
 <!--
-[JAR: get numbers from v3.0 when final; 
+[JAR: get numbers from v3.0 when final;
 `grep "^[a-z]" ../../feed/ott/edits/*.tsv | wc` = 97
 `cat amendments/*.json | grep original_label | wc` = 106
 `python util/count_patches.py amendments.py` = 121 ]
 -->
 
+## Assigning identifiers
+
 The final step is to assign unique, stable identifiers to nodes.  
```

Comment:
so that external links to OTT nodes will continue to functional correctly after the previous OTT version is replaced by the new one.